task1/src/GenreToggle.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+export function GenreToggle() {
```

Comment:
IMO - More elegant to use arrow functions =)

Reply:
const GenreToggle = () => {}

Reply:
fixed